liskovsubstitutionprinciple/Animal.java

```diff
@@ -0,0 +1,9 @@
+package solid.liskovsubstitutionprinciple;
```

Comment:
The same story with "interfacesegregationprinciple"

Reply:
Please check and change the others

Reply:
I have adjusted